<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicUploader</class>
 <widget class="QWidget" name="PicUploader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PicUploader</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="copyButton">
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="settingButton">
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
